Lib/fontTools/designspaceLib/statNames.py

```diff
@@ -113,8 +113,12 @@ def getStatNames(
 
     styleMapFamilyNames = {}
     for language in set(familyNames).union(styleNames.keys()):
-        familyName = familyNames.get(language, familyNames["en"])
-        styleName = styleNamesForStyleMap.get(language, styleNamesForStyleMap["en"])
+        try:
+            familyName = familyNames.get(language, familyNames["en"])
+            styleName = styleNamesForStyleMap.get(language, styleNamesForStyleMap["en"])
+        except KeyError:
+            LOGGER.warning("Cannot complete localized style map name because there are no English language fallbacks.")
```

Comment:
how likely is the situation that triggers this logging warning? if yes, it may deserve a test

Reply:
I think it happens if there's no default source _and_ you didn't set `en` names for something. I'll just remove it for now.